dev-util/tinlink/tinlink-1.0.0-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Tool to create very small elf binary from pure binary files"
+HOMEPAGE="http://sed.free.fr/tinlink/"
+SRC_URI="http://sed.free.fr/tinlink/${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+src_prepare() {
+	default
+	rm -f "${S}"/Makefile || die
```

Comment:
Why -f and shouldn't the CWD be S already?

Reply:
You're right, no need for -f and ${S}. - updated my PR